tests/e2e/specs/user-input-questions.test.js

```diff
@@ -36,66 +36,52 @@ import {
 	setSearchConsoleProperty,
 } from '../utils';
 
-const wpConditionalDescribe = () => {
-	// Only run for non WP 4.x versions
-	if ( process.env.WP_VERSION?.[ 0 ] !== '4' ) {
-		return describe;
-	}
-	return () => {
-		it( 'is skipped for WP 4.x', () => {} );
-	}; // no-op
-};
-
-wpConditionalDescribe()( 'User Input Settings', () => {
+describe( 'User Input Settings', () => {
 	async function fillInInputSettings() {
-		await step( 'select role', async () => {
-			await page.waitForSelector( '.googlesitekit-user-input__question' );
-			await expect( page ).toClick( '#role-owner_with_team' );
+		await page.waitForSelector(
+			'.googlesitekit-user-input__question--active'
```

Comment:
The added `--active` here is only really necessary I think due to the current layout which renders multiple questions at a time; it might also not be strictly needed but should help avoid trying to interact with elements of a non-active step.